### PR TITLE
Update for Step3-3

### PR DESCRIPTION
**Pull request by e-yang08**
## CHECKS :warning:

Please make sure you are aware of the following.

- [ ] **The changes in this PR doesn't have private information

### CODE REVIEW

**Review by KengoA — APPROVED** [review]
LGTM! You'll be working on using the database so this won't be necessary but maybe it's a good idea to first check if there's already an existing items.json file, leading to two cases when you set up a new database (json) or append to the existing one.

note: you might want to include the `__pycache__` directory to your .gitignore

**Comment by e-yang08** [comment]
> LGTM! You'll be working on using the database so this won't be necessary but maybe it's a good idea to first check if there's already an existing items.json file, leading to two cases when you set up a new database (json) or append to the existing one.
> 
> note: you might want to include the `__pycache__` directory to your .gitignore

Thank you so much for review!

- Updated `main.py` so that it checks if `items.json` file exists and initializes its format to account for the `post` endpoint where it requires dict key `items`.

- Updated `.gitignore` to ignore `__pycache__ directory`

File: python/main.py [hunk 1]
```diff
@@ -4,36 +4,85 @@
...
+            json.dump({"items": []}, file, indent=4)
+    logger.info("Created items.json if not initially exist")
+    return None
+
+
 @app.get("/")
 def root():
-    return {"message": "Hello, world!"}
+    return {"message": "Welcome to Mercari Items Database"}
+
```
Comment (manami-bunbun) [inline python/main.py:42]: Lovely!

**Review by manami-bunbun — APPROVED** [review]
LGTM!

**Pull request merged**